lib/tasks/bulk_email.rake

```diff
@@ -11,4 +11,25 @@ namespace :bulk_email do
       DeliveryRequestWorker.perform_async_in_queue(id, queue: :delivery_immediate)
     end
   end
+
+  desc "Email all subscribers of the Brexit checker"
+  task :brexit_subscribers, [:dry_run] => :environment do |_t, args|
+    brexit_list_ids = SubscriberList.where("subscriber_lists.tags->>'brexit_checklist_criteria' IS NOT NULL").pluck(:id)
+    email_ids =
+      brexit_list_ids.map do |id|
+        BulkSubscriberListEmailBuilder.call(
+          subject: ENV.fetch("SUBJECT"),
+          body: ENV.fetch("BODY"),
+          subscriber_lists: SubscriberList.where(id: id),
+        )
+      end
+    if args.dry_run == "not_dry_run"
```

Comment:
I would avoid this, as it will generate alerts about unsent emails, and they might accidentally get sent (since they exist in the system). How about just testing it on Integration or Staging, instead of a dry run feature?

Reply:
ok will do, thanks